cmd/commands/product-release.go

```diff
@@ -126,9 +158,15 @@ query getProductReleasesList(
 		},
 	}
 
-	productReleasesPage, err := client.GraphqlAPI[productReleaseSlice](ctx, cfConfig.NewClient().InternalClient(), query, variables)
+	productReleasesPage, err := client.GraphqlAPI[productReleaseSlice](ctx, cfConfig.NewClient().InternalClient(), latest_query, variables)
 	if err != nil {
-		return fmt.Errorf("failed to get product releases: %w", err)
+		if strings.Contains(err.Error(), "Cannot query field \\\"promotionCommits\\\" on type \\\"ProductReleaseStep\\\".") {
+			log.G().Warn("codefresh version v0.13.4 or older detected. Using v0.13.4 query which excludes promotionCommits.")
```

Comment:
I added this warning, but now I'm not sure as if there are customers/internal users using this cli command in automation the extra log line of output would make it fail.

I can also just make it fallback silently?

Reply:
i think that should be ok, showing a warning, but falling back to previous query.
this way they will know that something may work better if they upgrade their platform.